src/overlays/actors/ovl_Obj_Blockstop/z_obj_blockstop.c

```diff
@@ -28,12 +29,46 @@ const ActorInit Obj_Blockstop_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
+void ObjBlockstop_Init(Actor* thisx, GlobalContext* globalCtx) {
+    ObjBlockstop* this = THIS;
+    if (Flags_GetSwitch(globalCtx, (s32)this->actor.params)) {
+        Actor_MarkForDeath(&this->actor);
+    }
+    this->actionFunc = ObjBlockstop_CheckCollision;
```

Comment:
```suggestion
    ObjBlockstop* this = THIS;
    
    if (Flags_GetSwitch(globalCtx, (s32)this->actor.params)) {
        Actor_MarkForDeath(&this->actor);
    }
    this->actionFunc = ObjBlockstop_CheckCollision;
```
We like to keep a space between variables and the body of functions as part of the coding style.